hadoop-hdds/hadoop-metrics/src/main/java/org/apache/hadoop/ozone/metrics/AbstractMetric.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.metrics;
```

Comment:
Classes should be in their original package, `org.apache.hadoop.metrics2` and similar.  This lets Ozone immediately start using the code being forked.  Moving to Ozone-specific packages, which requires simple changes in both forked and original Ozone code, would be done in HDDS-12799.